core/src/main/java/psiprobe/beans/ContainerWrapperBean.java

```diff
@@ -19,16 +19,27 @@
 import org.apache.catalina.util.ServerInfo;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+
 import psiprobe.TomcatContainer;
 import psiprobe.model.ApplicationResource;
 
 /**
- * This class wires support for Tomcat "privileged" context functionality into Spring. If
- * application context is privileged Tomcat would always call servlet.setWrapper method on each
- * request. ContainerWrapperBean wires the passed wrapper to the relevant Tomcat container adapter
- * class, which in turn helps the Probe to interpret the wrapper.
+ * This class wires support for Tomcat "privileged" context functionality into
+ * Spring. If application context is privileged Tomcat would always call
+ * servlet.setWrapper method on each request. ContainerWrapperBean wires the
+ * passed wrapper to the relevant Tomcat container adapter class, which in turn
+ * helps the Probe to interpret the wrapper.
  */
+
+@Configuration
+@ComponentScan(basePackages = { "psiprobe" })
```

Comment:
@Aaron-LS I don't think we need to list componentScan here.  It's already loaded in ProveConfig and set to scan the entire base package so I think this is just duplicate and possibly might affect resource processing.